pytorch_lightning/metrics/metric.py

```diff
@@ -284,11 +284,17 @@ def persistent(self, mode: bool = False):
         for key in self._persistent.keys():
             self._persistent[key] = mode
 
-    def state_dict(self, *args, **kwargs):
+    def state_dict(self, destination=None, prefix='', keep_vars=False):
```

Comment:
I can see here https://github.com/pytorch/pytorch/blob/c7d348fea6ff4e20722256b38981c9c1f8727d95/torch/nn/modules/module.py#L1043-L1053 that `state_dict` is overloaded with two other signatures. Will this create problems?

Reply:
https://github.com/pytorch/pytorch/commit/eace0533985641d9c2f36e43e3de694aca886bd9#diff-4be56271f7bfe650e3521c81fd363da58f109cd23ee80d243156d2d6ccda6263R746-R759
It seems that they are relevant to the deletion of the generation logic for the pyi files.

Reply:
okay, sound good to me :]